doc/OnlineDocs/contributed_packages/mindtpy.rst

```diff
@@ -58,6 +59,21 @@ The solution may then be displayed by using the commands
 
   >>> SolverFactory('mindtpy').solve(model, mip_solver='glpk', nlp_solver='ipopt', tee=True)
 
+MindtPy also supports setting options for mip solver and nlp solver. 
+
+.. code::
+
+  >>> SolverFactory('mindtpy').solve(model, 
+                                     strategy='OA',
+                                     time_limit=3600,
+                                     mip_solver='gams',
+                                     mip_solver_args=dict(solver='gurobi', warmstart=True),
+                                     nlp_solver='gams',
+                                     nlp_solver_args=dict(solver='scip', warmstart=True, tee=True),
```

Comment:
I would use this example to also show the flexibility of MindtPy. Therefore, leave the MILP solver as GAMS + CPLEX and as NLP solver use IPOPT with an option for its maximum iterations to be changed.